### PR TITLE
Some typos

### PR DESCRIPTION
**Pull request by ionathanch**

### CODE REVIEW

**Comment by ionathanch** [comment]
@rzach A quick question: Where are the chapter summaries located (referenced to as `\OLEndChapterHook` in the TeX)? In the summary of chapter 5 of SLC there's a typo in the second sentence of the last paragraph: `A ⊧ ` should be ` ⊧ A`.

**Comment by rzach** [comment]
They're in https://github.com/rzach/sets-logic-computation
in the include directory

File: content/turing-machines/machines-computations/church-turing-thesis.tex [hunk 1]
```diff
@@ -10,15 +10,15 @@
...
-concept of an effective procedure. Turing took it that anyone who
-grasped the concept of an effective procedure and the concept of a
-Turing machine would have the intuition that anything that could be
-done via an effective procedure could be done by Turing machine. This
-claim is given support by the fact that all the other proposed precise
-replacements for the concept of an effective procedure turn out to be
-extensionally equivalent to the concept of a Turing machine---that is,
-they can compute exactly the same set of functions. This claim is
-called the \emph{Church-Turing thesis}.
+concept of an effective procedure. Turing took it to mean that anyone
```
Comment (rzach) [inline content/turing-machines/machines-computations/church-turing-thesis.tex:13]: What's meant here is that Turing thought that ... ("took it to mean that" requires a referent for "it" and it's not clear what that would be in this case)

Comment (ionathanch) [reply content/turing-machines/machines-computations/church-turing-thesis.tex:13]: So "Turing thought that anyone who grasped both..." should be good then?

**Comment by ionathanch** [comment]
Well I've reached the end of IC so that should be all the fixes I have. A few things to note:

* It seems like a big portion of the Lambda Calculus chapter has been rewritten/reorganized since the print version, so I've left most of that alone.
* In `second-order-arithmetic/metatheory/second-order-arithmetic.tex`, the proof of `sol-pa-definable` appears to be incomplete: the statement claims that + and × are definable in PA²† as well, but only ≤ is proven. I don't know if there was meant to be a proof, or if it was truncated on purpose for space, or if they're left as exercises.
* The commit with the most substantial changes was 5bdc40451ccbc3667e270751aca0f35b0d589c89, so that should probably be looked at more closely than the others.

@rzach You can review and merge etc. this now. Let me know if you need me to squash the commits or anything (although I think that can be done within GitHub).

**Comment by rzach** [comment]
Thanks for all your work!  I'll have to have a closer look at the second-order logic issues when I'm near my books next, but everything else looks fine.

**Pull request merged**